test/test_bonding.cpp

```diff
@@ -68,18 +78,18 @@ void listening_thread(bool should_read)
     ASSERT_EQ(inet_pton(AF_INET, "127.0.0.1", &bind_sa.sin_addr), 1);
     bind_sa.sin_port = htons(4200);
 
-    ASSERT_NE(srt_bind(server_sock, (sockaddr*)&bind_sa, sizeof bind_sa), -1);
+    ASSERT_NE(CHECK(srt_bind(server_sock, (sockaddr*)&bind_sa, sizeof bind_sa)), -1);
```

Comment:
This can be done more elegant with GTest:
```suggestion
    ASSERT_NE(srt_bind(server_sock, (sockaddr*)&bind_sa, sizeof bind_sa), -1) << "Last error: " << srt_getlasterror_str();
```

Reply:
Hmm, that would have to be applied to all others. Let me try something else then...